src/python/pants/backend/python/util_rules/pex_from_targets.py

```diff
@@ -150,30 +151,83 @@ def __init__(
         self.direct_deps_only = direct_deps_only
         self.description = description
 
-    @classmethod
-    def for_requirements(
```

Comment:
GitHub is showing a not-useful diff here. This method is simply gone, replaced by `get_requirements_pex` at the bottom of the file. _RelevantTargetsRequest is a different thing that happened to end up in this bit of the file...